x-pack/plugins/enterprise_search/public/applications/__mocks__/mount_async.mock.tsx

```diff
@@ -5,13 +5,13 @@
  * 2.0.
  */
 
+import { mountWithIntl } from './mount_with_i18n.mock';
```

Comment:
👍 similarly, I'm thinking I'll move most of these enzyme helpers outta `__mocks__`, honestly a `test_helpers` file might be better suited for these